packages/cli/src/__tests__/extension-gitlab-tracking.test.ts

```
@@ -40,12 +40,32 @@ vi.mock("@fusion/engine", () => ({
   createWorkflowAuthoringTools: vi.fn(() => ({})),
+  // FNXC:TestInfrastructure 2026-07-13-10:25: Complete the engine mock for extension.ts named imports (experiment finalize, workflow params, etc.).
+  defaultGitOps: {},
+  ExperimentFinalizeBranchExistsError: class MockError extends Error {},
+  ExperimentFinalizeCherryPickConflictError: class MockError extends Error {},
+  ExperimentFinalizeMergeBaseError: class MockError extends Error {},
+  ExperimentFinalizeNoKeptRunsError: class MockError extends Error {},
+  ExperimentFinalizePlanError: class MockError extends Error {},
+  ExperimentFinalizeService: vi.fn(),
+  ExperimentFinalizeStateError: class MockError extends Error {},
+  isInReviewMissingWorktreeSessionStartFailure: vi.fn(),
   workflowListParams: {},
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)